configure.ac

```diff
@@ -126,7 +126,6 @@ AC_PATH_PROG([NATURALDOCS], [naturaldocs], [notfound])
 AC_ARG_VAR([NATURALDOCS], [Path to naturaldocs])
 GLIB_COMPILE_RESOURCES=`$PKG_CONFIG --variable glib_compile_resources gio-2.0`
 AC_SUBST(GLIB_COMPILE_RESOURCES)
-AX_PYTHON_MODULE_VERSION([dbusmock], 0.10)
```

Comment:
Mmh, I didn't notice the file — and it did not get picked up either, because I had to install autoconf-archive in order to successfully pass the `autoreconf` phase.